rakelib/bootstrap.rake

```diff
@@ -1,2 +1 @@
-
-task "bootstrap" => [ "vendor:all", "compile:all" ]
+task "bootstrap" => [ "vendor:gems", "vendor:all", "compile:all" ]
```

Comment:
ah probably because you wanted to force the order before the jruby task. maybe its better to define the gems task as a dependency in the jruby task then?